README.md

```diff
@@ -61,39 +61,39 @@ If your notebook doesn't use the idea of epochs, then please add it to the varia
 If you have any other variables that would benefit by setting them to `1` during testing, add them to `strings_to_replace` in `runner.sh`.
 
 ### 4. List of notebooks and examples
-**2D classification**
-#### [mednist_tutorial](./2d_classification/mednist_tutorial.ipynb)
+#### <ins>**2D classification**</ins>
+##### [mednist_tutorial](./2d_classification/mednist_tutorial.ipynb)
 This notebook shows how to easily integrate MONAI features into existing PyTorch programs.
 It's based on the MedNIST dataset which is very suitable for beginners as a tutorial.
 This tutorial also makes use of MONAI's in-built occlusion sensitivity functionality.
 
-**2D segmentation**
-#### [torch examples](./2d_segmentation/torch)
+#### <ins>**2D segmentation**
```

Comment:
May I know why change the README format?
CC @wyli .

Thanks.

Reply:
making them headers and underlined to look better and make sharable link.. also making those headers capital case (some were in lower case and some in capital case)

you can see how it looks new vs current..
new: https://github.com/SachidanandAlle/tutorials/tree/nuclick#readme
current: https://github.com/Project-MONAI/tutorials/blob/main/README.md